api/cloudprotocol/envvars.go

```diff
@@ -36,14 +36,14 @@ const (
 // EnvVarsInstanceInfo struct with envs and related service and user.
 type EnvVarsInstanceInfo struct {
 	InstanceFilter
-	EnvVars []EnvVarInfo `json:"envVars"`
+	Variables []EnvVarInfo `json:"variables"`
 }
 
 // EnvVarInfo env info with id and time to live.
 type EnvVarInfo struct {
-	ID       string     `json:"id"`
```

Comment:
Why ID is remove?

Reply:
it is renamed to name according to public protocol